resources/did-configuration/index.html

```diff
@@ -648,7 +648,8 @@ <h2>
           <li>The object MUST contain an `id` property, and its value MUST be a valid DID URL reference.</li>
           <li>The object MUST contain a `type` property, and its value MUST be the string `"LinkedDomains"`.</li>
           <li>The object MUST contain an `origins` property, and its value MUST be an array of one or more 
```

Comment:
Its not immediately clear to me from reading this normative statement are we saying that origins must be only the domain where as service endpoint must be a valid HTTPs based URL? The linked resource albeit not contributed by this PR does not provide clarity around what we mean by `domain origins` (https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) infact searching the document reference returns no matches for `domain origins`

Reply:
perhaps a better link: https://tools.ietf.org/html/rfc6454

or
https://developer.mozilla.org/en-US/docs/Glossary/Origin ?